uber/site_sections/preregistration.py

```diff
@@ -134,6 +134,9 @@ def form(self, session, message='', edit_id=None, **params):
             attendee = session.attendee(params, ignore_csrf=True, restricted=True)
             group = session.group(group_params, ignore_csrf=True, restricted=True)
 
+        if c.BEFORE_PREREG_OPEN and not (c.IS_DEALER_REG_REQUEST and attendee.badge_type == c.PSEUDO_DEALER_BADGE):
```

Comment:
Huh, you had to add this to inside the function? Does that mean we're going to have to do that for the other portions of the `@check_if_can_reg` decorator?